packages/node/src/policy-gen/policy-gen-compartment-class.js

```diff
@@ -74,6 +73,11 @@ const updateModuleSource = (moduleDescriptor, canonicalName) => {
     throw new TypeError(`Unsupported module descriptor type; this is a bug`)
   }
 
+  // this just avoids adding duplicates. shouldn't happen, but who knows
+  if (moduleSource.imports.includes(canonicalName)) {
```

Comment:
(since `imports` is an _array_, we're not explicitly guaranteed unique items. a duplicate in this array would indicate a bug in `@endo/compartment-mapper`, since we're explicitly avoiding the case here)